Signal/src/AppDelegate.m

```diff
@@ -52,6 +53,23 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     CategorizingLogger *logger = [CategorizingLogger categorizingLogger];
     [logger addLoggingCallback:^(NSString *category, id details, NSUInteger index){
     }];
+    
+    BOOL loggingIsEnabled;
+    
+#ifdef DEBUG
+    // Specified at Product -> Scheme -> Edit Scheme -> Test -> Arguments -> Environment to avoid things like
```

Comment:
What can you specify in the scheme that affects the code here? I feel like this comment is referring to code that no longer exists. Or does it make sense to you?